### DISCUSSION TITLE
How do I download quarantined files?

### DISCUSSION THREAD

**Discussion by cosrah** [CrowdStrike/falconpy#878] [Q&A] [answered]
I already checked Python SDK in GitHub, but I can't see the API for **downloading the quarantined file**.

Is there any way?

Best regards,

**Comment by jshcodes** [comment, marked as answer]
Hi @cosrah!

Quarantine files can now be downloaded via the Sandbox using the [Quarantine API](https://www.falconpy.io/Service-Collections/Quarantine.html).

We have a [sample available here](https://github.com/CrowdStrike/falconpy/tree/main/samples/quarantine#get-quarantine-files) demonstrating how to download all quarantined files within your environment.
